Update document and comment (#2304)
infiniflow/infinity: docs/references/configurations.mdx

# The maximum number of connections. Defaults to 256.
# Range: [1, 65536]
connection_pool_size     = 128

> # IP address of the Infinity peer server, accessed by peer Infinity node

Reviewer: ```suggestion
# The IP address of the Infinity peer server to be accessed by a peer node
```

peer_ip                  = "0.0.0.0"

> # Port of the Infinity peer server, accessed by peer Infinity node

Reviewer: ```suggestion
# The port of the Infinity peer server to be accessed by a peer node
```

peer_port                = 23850

> # If peer server can't connect, retry delay time

Reviewer: ```suggestion
# The delay time for reconnecting to the Infinity peer server after a failed connection
```

peer_retry_delay         = 0

> # If peer server can't connect, retry count

Reviewer: ```suggestion
# The maximum number of reconnection attempts to the Infinity peer server after a failed connection
```

peer_retry_count           = 0

> # If peer server connect timeout

Reviewer: ```suggestion
# Timeout for connecting to the Infinity peer server
```

peer_connect_timeout     = 2000

> # If peer server data receive timeout

Reviewer: ```suggestion
# Timeout for the Infinity peer server to receive data
```

peer_recv_timeout        = 0

> # If peer server data send timeout

Reviewer: ```suggestion
# Timeout for the Infinity peer server to send data
```
